src/modules/todo/hooks/useFetchTodos.ts

```diff
@@ -0,0 +1,11 @@
+import { useQuery } from '@tanstack/react-query'
+import { TodoApiService } from "../api"
+
+export function useFetchTodos() {
+    const { isLoading, data } = useQuery({
+        queryKey: ['todos'],
```

Comment:
It's better to put key into constant in this file, for example: `export const FETCH_TODO_LIST_QUERY_KEY = 'todos'` resoning behind this is that you might need this key to invalidate query or interact with query in a different way and query key ensures that you will use correct query, as if it's would be just string you might misspell it, those bugs is pretty hard to catch, so better to use constants.